docs/stable/1-client.md

```diff
@@ -165,9 +171,13 @@ It uses Ethereum Node Records (ENR) to report discovered peers.
 Both peers discovery mechanisms use topics to provide peers with certain capabilities.
 There is no point in returning peers that do not support a particular protocol.
 Status nodes that want to be discovered MUST register to Discovery V5 and/or Rendezvous
-with the `whisper` topic. Status nodes that are mail servers and want to
+with the `waku` topic. Status nodes that are mail servers and want to
```

Comment:
See above, MUST seems like too strong of a word and would also make this spec incompatible with v1 in wild, which is not what we want / are doing

Reply:
Ok. Thank you.